make.cmd

```diff
@@ -18,6 +18,16 @@ DEL /F /A /Q y.output
 
 golex -o parser/scanner.go parser/scanner.l
 
+@echo [Ast-Parser]
+go get github.com/qiuyesuifeng/goyacc
+go get github.com/qiuyesuifeng/golex
+type nul >>temp.XXXXXX & copy temp.XXXXXX +,,
```

Comment:
I saw the doc:
If you want to assign the current time and date to a file without modifying the file, use the following syntax:
copy /b Source+,,
The commas indicate the omission of the Destination parameter.

but why?